src/main/java/Storage.java

```diff
@@ -0,0 +1,154 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+class Storage {
+
+    private File f;
+    private static final String path = "src/main/data/userTasks.txt";
```

Comment:
Should this be in uppercase instead since you are declaring a constant?

```suggestion
    private static final String PATH = "src/main/data/userTasks.txt";
```

Reply:
Right, I guess I missed this out. Thank you for your comment!